include/picongpu/plugins/openPMD/writer/ParticleAttribute.hpp

```diff
@@ -115,39 +117,23 @@ namespace picongpu
 
                     if(elements == 0)
                     {
-                        params->openPMDSeries->flush(PreferredFlushTarget::Disk);
                         continue;
                     }
 
                     ValueType* dataPtr = frame.getIdentifier(Identifier()).getPointer(); // can be moved up?
                     // ask openPMD to create a buffer for us
-                    // in some backends (ADIOS2), this allows avoiding memcopies
-                    auto span = recordComponent
-                                    .storeChunk<ComponentType>(
-                                        ::openPMD::Offset{globalOffset},
-                                        ::openPMD::Extent{elements},
-                                        [&storeBfr](size_t size)
```

Comment:
The storeBfr is defined above and can now be deleted too. Will look in further detail later or tomorrow, including tests.

Reply:
thx, done